### PR TITLE
Improves Labour Camp

### PR DESCRIPTION
**Pull request by FloranOtten**
[Changelogs]: 

:cl: Gulag is better now, comrade
add: Added an atmospherics system to the Labour Camp
add: Added a better Security Office to Labour Camp
change: Changes exterior Gulag door to actual airlocks
/:cl:

[why]: The Labour Camp has always felt a bit haphazard to me. I hope this will if not fix at least improve it. Also insert_depressurized_gulag_meme

Yes I am aware my logs are a mess

### CODE REVIEW

**Comment by 81Denton** [comment]
Could you post some screenshots?

**Comment by FloranOtten** [comment]
https://imgur.com/gallery/wHzhxcG

**Comment by HydroFloric** [comment]
does this effect people being able to escape by crossing the lava?

**Comment by BeeSting12** [comment]
Looks good, 👍

**Comment by FloranOtten** [comment]
> does this effect people being able to escape by crossing the lava?

It does not, you'd still have to sprint across a lava moat.

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -3372,6 +3419,20 @@
 	},
 /turf/open/indestructible/boss,
 /area/lavaland/surface/outdoors)
+"nJ" = (
+/obj/structure/lattice/catwalk,
+/obj/machinery/atmospherics/components/unary/outlet_injector/on{
+	icon_state = "inje_map";
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: dv

File: _maps/map_files/Mining/Lavaland.dmm [hunk 2]
```diff
@@ -898,8 +930,11 @@
 	},
 /area/mine/production)
 "cG" = (
-/obj/effect/spawner/structure/window/reinforced,
-/turf/open/floor/plating,
+/obj/machinery/computer/secure_data{
+	icon_state = "computer";
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: dv

File: _maps/map_files/Mining/Lavaland.dmm [hunk 3]
```diff
@@ -874,6 +902,10 @@
 /area/mine/laborcamp)
 "cB" = (
 /obj/machinery/portable_atmospherics/canister/air,
+/obj/machinery/atmospherics/components/unary/portables_connector/visible{
+	icon_state = "connector_map";
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: dv

File: _maps/map_files/Mining/Lavaland.dmm [hunk 4]
```diff
@@ -195,8 +195,10 @@
 /obj/effect/mapping_helpers/airlock/cyclelink_helper{
 	dir = 4
 	},
-/obj/machinery/door/airlock{
-	name = "Labor Camp External Access"
+/obj/machinery/door/airlock/external{
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: /obj/machinery/door/airlock/external/glass

File: _maps/map_files/Mining/Lavaland.dmm [hunk 5]
```diff
@@ -3678,6 +3779,13 @@
 /obj/machinery/atmospherics/pipe/simple/supply/hidden,
 /turf/open/floor/plating,
 /area/mine/living_quarters)
+"RC" = (
+/obj/machinery/computer/prisoner{
+	icon_state = "computer";
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: dv

File: _maps/map_files/Mining/Lavaland.dmm [hunk 6]
```diff
@@ -3782,8 +3897,10 @@
 /obj/effect/mapping_helpers/airlock/cyclelink_helper{
 	dir = 8
 	},
-/obj/machinery/door/airlock{
-	name = "Labor Camp External Access"
+/obj/machinery/door/airlock/external{
```
Comment (ShizCalev) [inline _maps/map_files/Mining/Lavaland.dmm]: /obj/machinery/door/airlock/external/glass

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**